subprojects/docs/src/docs/userguide/publishing_maven.adoc

```diff
@@ -106,6 +106,24 @@ include::sample[dir="signing/maven-publish/groovy",files="build.gradle[tags=pom-
 include::sample[dir="signing/maven-publish/kotlin",files="build.gradle.kts[tags=pom-customization]"]
 ====
 
+[[publishing_maven:resolved_dependencies]]
+=== Customizing dependencies versions
+
+By default, the set of dependencies and constraints that are added to the POM file will contain the versions _declared_ in the build file.
+Sometimes it is desirable to publish the _resolved_ version instead.
```

Comment:
I think it's worth explaining the "sometimes" here. Use cases is what people want to read, I think.

Reply:
I have added two use cases, feel free to suggest additional ones.